src/main/java/com/google/sps/servlets/GroupsServlet.java

```diff
@@ -57,18 +58,21 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         ImmutableList.Builder<ImmutableMap<String, String>> groupsListBuilder = new ImmutableList.Builder<>();
```

Comment:
I can't leave at comment at unchanged code, but at the top it says TODO: Unhardcode the groups, do you want to leave that hardcoded group as the default group that everyone is automatically join to?

Reply:
i removed it in my PR so it should be dealt with once it's mergomg!